ionicons/IonIcons.m

```diff
@@ -19,10 +19,33 @@ @implementation IonIcons
 + (UIFont*)fontWithSize:(CGFloat)size;
 {
     UIFont* font = [UIFont fontWithName:@"ionicons" size:size];
+    if (!font) {
+        [[self class] dynamicallyLoadFontNamed:@"ionicons"];
```

Comment:
Good point, I think yours is safer in that regard since the font stuff does have an unregister call, I was just looking for a way to avoid some conditional logic. I'll use your original version and change the assert message below & the documentation.